extension/package.json

```diff
@@ -64,7 +65,7 @@
     "prettier": "2.2.1",
     "proxyquire": "^2.1.3",
     "puppeteer": "^5.5.0",
-    "sinon": "^9.2.4"
+    "sinon": "^9.2.3"
```

Comment:
downgrading versions why?

Reply:
Didn't modify it all the time.
It upgrades in multitenancy branch after I rebase.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/528/commits/fab9d4ec5a6c045546afc565c3443fac53ea4447